Misc/NEWS.d/next/Core and Builtins/2024-01-12-16-40-07.gh-issue-113939.Yi3L-e.rst

```diff
@@ -0,0 +1,7 @@
+frame.clear(), clear locals, not just fast locals. This is relevant once
```

Comment:
Can you reword this to be a clear description of what the change in this PR is?  The first sentence in particular is hard to understand.

Reply:
I reworded this.